install.sh

```diff
@@ -1,7 +1,78 @@
 #!/bin/bash
 
+#repo addresses
+aasdkRepo="https://github.com/OpenDsh/aasdk"
+gstreamerRepo="git://anongit.freedesktop.org/gstreamer/qt-gstreamer"
+openautoRepo="https://github.com/rhysmorgan134/openauto.git"
+
+#Help text
+display_help() {
+    echo
+    echo "   --deps           install all dependencies"
+    echo "   --aasdk          install and build aasdk"
+    echo "   --openauto       install and build openauto "
+    echo "   --gstreamer      install and build gstreamer "
+    echo "   --dash           install and build dash "
+    echo
+    exit 1
+}
+
+#location of OS details for linux
+OS_RELEASE_FILE="/etc/os-release"
+
+#check if Raspian is in the file, if not set the install Args to be false
+if grep -q "Raspbian" ${OS_RELEASE_FILE}; then
```

Comment:
@robert5974 might be able to weigh in here, is the new Raspberry OS still internally named "Raspbian"?

Reply:
When I check for an update it still says Raspbian Buster.  That is with a full dist-upgrade from Buster and also a clean flash using the raspberry pi flasher utility which will auto pull the latest is image.